### PR TITLE
some cython-linting in modular/ folder

### PR DESCRIPTION
**Pull request by fchapoton**
### :books: Description

This fixes some of the issues found by cython-lint in the modular folder. The command used was
```
cython-lint --ignore=E501,E265,E231,E127,E266,E741 src/sage/modular/
```

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/modular/arithgroup/congroup.pyx [hunk 1]
```diff
@@ -204,29 +205,28 @@ def degeneracy_coset_representatives_gamma1(int N, int M, int t):
...
     """
-
     if N % M != 0:
-        raise ArithmeticError("M (=%s) must be a divisor of N (=%s)" % (M,N))
+        raise ArithmeticError(f"M (={M}) must be a divisor of N (={N})")
 
-    if (N/M) % t != 0:
-        raise ArithmeticError("t (=%s) must be a divisor of N/M (=%s)"%(t,N/M))
+    if (N // M) % t != 0:
+        raise ArithmeticError("t (={t}) must be a divisor of N/M (={N//M})")
```
Comment (mkoeppe) [inline src/sage/modular/arithgroup/congroup.pyx]: f"..."

Comment (fchapoton) [reply src/sage/modular/arithgroup/congroup.pyx]: indeed, sorry. Fixed now

**Review by mkoeppe — APPROVED** [review]

**Comment by dcoudert** [comment]
You have not considered the comments I made, but it can be done in a future ticket.

**Comment by fchapoton** [comment]
> You have not considered the comments I made, but it can be done in a future ticket.

heu, pardon, mais quels commentaires ?

**Pull request merged**